packages/governance/src/closingRule.js

```diff
@@ -10,7 +10,7 @@ import { Far } from '@endo/marshal';
 /** @type {CloseVoting} */
 export const scheduleClose = (closingRule, closeVoting) => {
   const { timer, deadline } = closingRule;
-  E(timer).setWakeup(
+  return E(timer).setWakeup(
```

Comment:
```suggestion
  void E(timer).setWakeup(
```

`CloseVoting` claims to have return type `void`